### PR TITLE
Flag users as "Always premoderate"

### PR DESCRIPTION
**Pull request by gerardogalvez**
## What does this PR do?
Issue #2030  
- Status can be set using the action dropdown in the People tab and in User Details.
- Users flagged as "Always premoderate" will have their comments sent to the premod queue.
- Users can be filtered with the Always Premoderate status.
- Include Always Premoderate status changes in User History.
- Show Always Premoderate status in Users CLI.
- Add spanish translations.

## How do I test this PR?
1. Go to the People tab and select Always Premoderate User from the actions dropdown (or from User Details)
2. Confirm in the dialog.
3. Log in as the premoderated user and write a comment.
4. Go to the Premod queue and the comment should be there.

### CODE REVIEW

**Comment by kgardnr** [comment]
Thank you @gerardogalvez! I did a quick test and it looks good. I'll pass it onto my teammates for design + code review and we'll get back to you! Awesome work though 🔥

**Review by kgardnr — APPROVED** [review]

**Comment by immber** [comment]
@gerardogalvez This PR is failing our integration tests with the following error... 
```
Timed out while waiting for element <Section[name=suspendUserDialog],Element[name=@step0]> to be visible for 10000 milliseconds.  - expected "visible" but got: "not visible"
    at Object.admin suspends user (/home/circleci/coralproject/talk/test/e2e/specs/06_suspendUser.js:105:8)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)

Error processing the server response: 
 <html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 500 Server Error</title>
</head>
<body><h2>HTTP ERROR 500</h2>
<p>Problem accessing /wd/hub/session/null/log. Reason:
<pre>    Server Error</pre></p><h3>Caused by:</h3><pre>java.lang.IllegalArgumentException: Cannot find constructor to populate
	at org.openqa.selenium.remote.server.AllHandlers.lambda$create$9(AllHandlers.java:156)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at org.openqa.selenium.remote.server.AllHandlers.create(AllHandlers.java:156)
	at org.openqa.selenium.remote.server.AllHandlers.lambda$handler$3(AllHandlers.java:140)
	at org.openqa.selenium.remote.server.AllHandlers.lambda$match$0(AllHandlers.java:85)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
	at org.openqa.selenium.remote.server.AllHandlers.match(AllHandlers.java:87)
	at org.openqa.selenium.remote.server.WebDriverServlet.handle(WebDriverServlet.java:201)
	at org.openqa.selenium.remote.server.WebDriverServlet.doPost(WebDriverServlet.java:160)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at org.openqa.selenium.remote.server.WebDriverServlet.service(WebDriverServlet.java:120)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:841)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:535)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.seleniumhq.jetty9.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:190)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:561)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:334)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.seleniumhq.jetty9.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.seleniumhq.jetty9.io.FillInterest.fillable(FillInterest.java:104)
	at org.seleniumhq.jetty9.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:247)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:140)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
	at org.seleniumhq.jetty9.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:243)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:679)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:597)
	at java.lang.Thread.run(Thread.java:748)
</pre>
<hr><a href="http://eclipse.org/jetty">Powered by Jetty:// 9.4.7.v20170914</a><hr/>

</body>
</html>

Error: Test Case failed, skipping all the rest
    at Object.afterEach (/home/circleci/coralproject/talk/test/e2e/specs/06_suspendUser.js:13:13)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
It looks like this change is affecting the display of the `suspendUserDialog` UI components, which is causing the tests to fail. 

We should also consider adding test cases for this new functionality before merging this feature in.

**Comment by kgardnr** [comment]
@immber that error might be on the Circle side actually, seems our builds were breaking but they seem to be working again - we'll see if it happens again!

File: client/coral-admin/src/components/UserDetail.css [hunk 1]
```diff
@@ -149,6 +149,12 @@
   color: white;
 }
 
+.actionsMenuAlwaysPremod {
```
Comment (alinhle) [inline client/coral-admin/src/components/UserDetail.css]: If you move this above line `140: .actionsMenuSuspended`, then _suspension_ and _banned_ styles will have higher priority than _alwaysPremod_

File: client/coral-admin/src/routes/Community/components/People.css [hunk 2]
```diff
@@ -142,6 +142,12 @@ th.header:nth-child(2), th.header:nth-child(3) {
   color: white;
 }
 
+.actionsMenuAlwaysPremod {
```
Comment (alinhle) [inline client/coral-admin/src/routes/Community/components/People.css]: See above

**Review by alinhle — CHANGES_REQUESTED** [review]
@gerardogalvez Apologies that it took so long for me to get to this. I've reviewed the front-end part and you've done a pretty good job! I only have a small comment for you. @wyattjoh is going to look at the server part pretty soon too. Thanks for your amazing work!

**Comment by kgardnr** [comment]
Hey @gerardogalvez just a few comments from our team and then we can merge! Thank you so much for this! It's awesome <3

**Review by wyattjoh — APPROVED** [review]

**Comment by mkonikov** [comment]
@kgardnr Did you just need a change to the CSS order now? Something I can help with?

**Comment by kgardnr** [comment]
@mkonikov yes, it's just the 2 comments @cvle had above. If you wanna submit those, we can merge this!

**Comment by mkonikov** [comment]
FWIW made PR to @gerardogalvez's repo with those changes here: https://github.com/gerardogalvez/talk/pull/1/files

Would need @gerardogalvez to approve and merge.

**Comment by gerardogalvez** [comment]
@mkonikov Merged, thanks! 😄

**Review by alinhle — APPROVED** [review]

**Comment by alinhle** [comment]
Amazing thanks @gerardogalvez and @mkonikov

**Pull request merged**